packages/babel-parser/test/helpers/runFixtureTests.js

```diff
@@ -127,15 +129,21 @@ function overrideToJSON(cb) {
     const { toJSON } = obj.prototype;
     originalToJSONMap.set(obj, toJSON);
     obj.prototype.toJSON = function() {
+      if (typeof this === "bigint") {
+        return { [serialized]: "BigInt", value: serialize(this) };
```

Comment:
This serialization using a "type annotation" is needed because, in envs which don't support BigInts, we must know if `"1"` represented an integer (which should thus be compared as if it was `null`) or the string `"1"`.

I will expand this serialization to regular expressions, but in a separate PR to keep this one as small as possible so that it can be merged quickly (and thus we can rely on parser tests again).